src/cocotb/task.py

```diff
@@ -233,6 +242,35 @@ def exception(self) -> Optional[BaseException]:
         else:
             return None
 
+    def add_done_callback(self, callback: Callable[["Task[ResultType]"], Any]) -> None:
+        """Add a callback to run once the Task becomes "done".
+
+        Args:
+            callback: The callback to run on "done".
```

Comment:
```suggestion
            callback: The callback to add to the "run-on-done" list.
```

Reply:
I think specifying `"run-on-done"` is maybe a bit too flowery. I used https://docs.python.org/3/library/asyncio-task.html#asyncio.Task.add_done_callback as guidance.